pkg/fsm/handler/observed_event_handler_test.go

```diff
@@ -231,14 +232,14 @@ func TestObserveEnqueueOwner(t *testing.T) {
 				log,
 				scheme,
 				controllerName,
-				metrics,
+				m,
 				handler.EnqueueRequestForOwner(scheme, testrestmapper.TestOnlyStaticRESTMapper(scheme), &corev1.ConfigMap{}),
 				fsmhandler.TriggerTypeChild,
 			)
 		}
 
 		t.Run(tc.name, func(t *testing.T) {
-			queue := workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())
```

Comment:
The new Typed Default controller rate limiter has a delay of 5ms https://github.com/kubernetes/client-go/blob/e421aed7f1b4515185aecebbe101dcfa5fd52997/util/workqueue/default_rate_limiters.go#L50

Reply:
What error results from the tests if we don't sleep? Trying to understand the exact mechanics